css/styles.css

```diff
@@ -0,0 +1,288 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Please use bootstrap classes when possible.

Reply:
added margin and padding to body tag